lib/models/Order.js

```diff
@@ -11,21 +13,58 @@ module.exports = class Order {
 
   static async insert({ product, quantity }) {
     // TODO: Implement me
+    const { rows } = await pool.query(
+      `INSERT INTO 
+        orders(product, quantity) 
+      VALUES 
+        ($1, $2) 
+      RETURNING 
+        *;`,
+      [product, quantity]
+    );
+    return new Order(rows[0]);
   }
 
   static async getAll() {
-    // TODO: Implement me
+    const { rows } = await pool.query(
+      `SELECT 
+        * 
+      FROM 
+        orders;
+    `,
+      []
+    );
+    return rows;
```

Comment:
Eventually, we'll  cover mapping through the rows for getAlls like so: `return rows.map((row) => new Order(row));`